modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheOperationContext.java

```diff
@@ -31,9 +31,12 @@
  */
 public class CacheOperationContext implements Serializable {
     /** */
-    //TODO IGNITE-8801 remove this and set default as `false`.
-    public static final boolean DFLT_ALLOW_ATOMIC_OPS_IN_TX =
-        IgniteSystemProperties.getBoolean(IGNITE_ALLOW_ATOMIC_OPS_IN_TX, true);
+    private static final boolean DFLT_ALLOW_ATOMIC_OPS_IN_TX = false;
+
+    /** */
+    public static final boolean allowAtomicOpsInTx() {
```

Comment:
The method name should be changed because we already have a method called `setAllowAtomicOpsInTx`, but it is not related to the static one. How about `isAtomicOpsAllowInTxByDefault`

Reply:
Renamed to `defaultAllowAtomicOpsInTx()`